util.go

```diff
@@ -383,3 +389,9 @@ func go_set_authorizer(data unsafe.Pointer, op C.int, arg1, arg2, db, entity *C.
 	fn := authorizerRegistry.lookup(idx).(AuthorizerFunc)
 	return C.int(fn(int(op), raw(goStr(arg1)), raw(goStr(arg2)), raw(goStr(db)), raw(goStr(entity))))
 }
+
+//export go_error_log
+func go_error_log(_ unsafe.Pointer, errCode C.int, msg *C.char) {
+	fn := errorLogCallback.Load().(ErrorLogFunc)
+	fn(int(errCode), C.GoString(C.sqlite3_errstr(errCode)), C.GoString(msg))
```

Comment:
This could potentially be changed to

```suggestion
	fn(int(errCode), raw(goStr(C.sqlite3_errstr(errCode))), raw(goStr(msg)))
```

like in `go_set_authorizerer` etc. above (would also require adjusting the signature of `ErrorLogFunc`).

I opted against this because (1) I didn't feel 100% confident that it would be correct, (2) any performance gain would be negligible, since these error/warning log events are not expected to be frequent occurrences normally.